static_frame/core/index_hierarchy.py

```diff
@@ -1638,7 +1674,7 @@ def level_drop(self,
                 blocks = self._blocks.iloc[NULL_SLICE, :count]
                 return self.__class__(levels,
                         blocks=blocks,
-                        own_blocks=True,
+                        own_blocks=True, # Should this be false?
```

Comment:
We can own the blocks here as a the `self._blocks.iloc` call above gave us a new `TypeBlocks` instance.